webapp/src/main/webapp/js/mc-view.js

```diff
@@ -160,93 +142,95 @@ MonitoringConsole.View = (function() {
+        const pullAvailable = !MonitoringConsole.Model.Role.isGuest();
+        return [
+            { id: 'settings-general', type: 'app', caption: 'General', entries: [
+                { label: 'Data Refresh', input: [
+                    { type: 'value', unit: 'sec', value: MonitoringConsole.Model.Refresh.interval(), onChange: (val) => MonitoringConsole.Model.Refresh.interval(val) },
+                    { type: 'toggle', options: { false: 'Pause', true: 'Play'}, value: !MonitoringConsole.Model.Refresh.isPaused(), onChange: (checked) => MonitoringConsole.Model.Refresh.paused(!checked) },
+                ]},
+                { label: 'Page Rotation', input: [
+                    { type: 'value', unit: 'sec', value: MonitoringConsole.Model.Settings.Rotation.interval(), onChange: (val) => MonitoringConsole.Model.Settings.Rotation.interval(val) },
+                    { type: 'toggle', options: { false: 'Off', true: 'On' }, value: MonitoringConsole.Model.Settings.Rotation.isEnabled(), onChange: (checked) => MonitoringConsole.Model.Settings.Rotation.enabled(checked) },
+                ]},
+                { label: 'Role', input: [
+                    { label: MonitoringConsole.Model.Role.name() },
+                    { input: () => Components.createIconButton({
+                        class: 'btn-icon',
+                        icon: 'icon-pencil',
+                        alt: 'Change User Role...'
+                    }).click(showRoleSelectionModalDialog) },
+                ]},
+                { label: 'Watches', input: $('<button/>').text('Go to Watch Settings').click(showWatchConfigModalDialog) },
             ]},
-            { label: 'Role', type: 'dropdown', options: {guest: 'Guest', user: 'User', admin: 'Administrator'}, value: MonitoringConsole.Model.Role.get(), onChange: (val) => { MonitoringConsole.Model.Role.set(val); updateSettings(); } },
-            { label: 'Page Sync', available: pushAvailable || pullAvailable, input: [
-                { available: pushAvailable, input: () => $('<button />', { text: 'Update Remote Pages', title: 'Push local state of all know remote pages to server'}).click(MonitoringConsole.Model.Page.Sync.pushAllLocal) },
-                { available: pullAvailable, input: () => $('<button/>', { text: 'Update Local Pages', title: 'Open Page synchronisation dialoge'}).click(onPagesSync) }, 
+            { id: 'settings-pages', available: pushAvailable || pullAvailable, type: 'app', caption: 'Pages', entries: [
+                { label: 'Server Sync', input: [
+                    { available: pushAvailable, input: () => $('<button />', { text: 'Update Server...', title: 'Push local state of all know remote pages to server'}).click(showPagePushModalDialog) },
+                    { available: pullAvailable, input: () => $('<button/>', { text: 'Update Local...', title: 'Open Page synchronisation dialoge'}).click(showPageSyncModalDialog) }, 
```

Comment:
```suggestion
                    { available: pullAvailable, input: () => $('<button/>', { text: 'Update Local...', title: 'Open Page synchronisation dialogue'}).click(showPageSyncModalDialog) }, 
```
Typo

Reply:
Separately, might want to think about the capitalisation
Is Page a noun in this instance?
Should we be using Title Case?

Reply:
Fixed this one manually as I also change `Page` to `page`.